setup.py

```diff
@@ -38,6 +38,21 @@ def get_version(rel_path):
     #extra dependency, only needed if minizinc is to be used.
```

Comment:
Perhaps the github cli config can also be updated to re-use the optional dependencies

Reply:
This is quite separate, so doesn't have to be done here. If I have time, I'll do it for this release https://github.com/CPMpy/cpmpy/issues/617